### PR TITLE
git-project should not crash when there's an empty line at the end of the .gitproj file

### PR DESCRIPTION
**Pull request by Yxwww**

### CODE REVIEW

**Comment by Yxwww** [comment]
WOW this PR is from one year ago. see if we can get this in.

File: Makefile [hunk 1]
```diff
@@ -34,5 +34,7 @@ uninstall:
 	    rm $(LOCALPATH)/$(EXE); \
 	fi
 
-tests:
+reinstall: uninstall install
+
+tests: reinstall
```
Comment (bsmithyman) [inline Makefile]: This seems odd; why are we running the `reinstall` target when running `make tests`?

**Comment by Yxwww** [comment]
Hey @bsmithyman  thanks for taking a look. 
It seems the tests actually relies on testing the `git project` command we installed to the environment. 
So I was thinking running `reinstall` before test will ensure the tests always up to date with the repo. 

Not sure if this makes sense tho. Or there can be other alternative approach ?

**Review by bsmithyman — APPROVED** [review]
Thanks for bringing this to my attention @Yxwww! Looks good to get in :)

**Pull request merged**